dynawo/sources/Modeler/Common/DYNModelMulti.cpp

```diff
@@ -1150,13 +1150,14 @@ void ModelMulti::printVariableNames() {
   }
   nVar = 0;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
-  Trace::debug(Trace::variables()) << "X variables" << Trace::endline;
+  Trace::debug(Trace::variables()) << "X variables (with initial type)" << Trace::endline;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
+  const std::vector<propertyContinuousVar_t>& modelYType = getYType();
   for (std::vector<boost::shared_ptr<DYN::SubModel> >::const_iterator it = subModels_.begin(); it != subModels_.end(); ++it) {
     const std::vector<std::string>& xNames = (*it)->xNames();
     for (unsigned int j = 0; j < xNames.size(); ++j) {
       std::string varName = (*it)->name() + "_" + xNames[j];
-      Trace::debug(Trace::variables()) << nVar << " " << varName << Trace::endline;
+      Trace::debug(Trace::variables()) << nVar << " " << varName << " " << propertyVar2Str(modelYType[nVar]) << Trace::endline;
```

Comment:
Do we want to keep this change as I think it's not used in the final version of the delay PR?

Reply:
If I remember well the issue is that when we call this the dynamic types are not initialized yet

Reply:
Yes it was the issue, as it is a log file I just printed twice once before calculateIC and once after with the new info when types are available. I think we don't really care about writing it twice as it should never be called most of the time